server/db/models/user.js

```diff
@@ -18,6 +18,7 @@ const User = db.define('user', {
       return () => this.getDataValue('password')
     }
   },
+
```

Comment:
are these necessary? remove if not

Reply:
I believe this is necessary, they discuss it in the setup video as a workaround for the lack of private variables. In other words, putting the password functionality as a method on the model makes it private

Reply:
oh, you mean the extra spacing between model fields, they've been removed